lib/spack/spack/build_systems/python.py

```diff
@@ -222,6 +221,25 @@ def install(self, spec, prefix):
 
         self.setup_py('install', *args)
 
+    @run_after('install')
+    def write_python_sitedirs_file(self):
+
+        spec = self.spec
+        mkdirp(spec.prefix.bin)
+
+        sitedirs = []
+        sitedirs.append(spec.prefix)
+
+        deps = spec.dependencies(deptype=('link', 'run'))
+        my_python = next( (spec for spec in deps if spec.satisfies('python')), None)
```

Comment:
you should probably add a comment here to explain what happens here

Reply:
done.